src/main/java/core/basesyntax/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawFigure {
```

Comment:
```suggestion
public interface Drawable {
```
Not only Figure can implement it.